.github/workflows/extract-translation-source-files.yml

```diff
@@ -125,8 +153,14 @@ jobs:
           fi
           # remove translations/${{ matrix.repo }}/.git so we don't commit a submodule
           rm -rf .git
+
+      # All repositories except for edx-platform follows the standard Django i18n pattern
+      - name: add repo translation source files openedx-translations
+        if: matrix.repo != 'edx-platform'
```

Comment:
This is no longer the case. The platform should be standardized. Please make a pull request in the `edx-platform` to use `--no-segment`.

```suggestion
```